bespin_api_v2/serializers.py

```diff
@@ -0,0 +1,64 @@
+from rest_framework import serializers
+from django.contrib.auth.models import User
+from data.models import Workflow, WorkflowVersion, VMStrategy, WorkflowConfiguration
+import json
+
+# This field can go away when we switch to JSONField
+class JSONStrField(serializers.Field):
+    def to_representation(self, value):
+        return json.loads(value)
+
+    def to_internal_value(self, data):
+        return json.dumps(data)
+
+
+class AdminWorkflowSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Workflow
+        resource_name = 'workflows'
+        fields = '__all__'
+
+
+class AdminWorkflowVersionSerializer(serializers.ModelSerializer):
+    fields = JSONStrField(source="fields_json")
+
+    class Meta:
+        model = WorkflowVersion
+        resource_name = 'workflowversions'
+        fields = ['id', 'workflow', 'description', 'object_name', 'created', 'version', 'url', 'fields']
+
+
+class WorkflowConfigurationSerializer(serializers.ModelSerializer):
+    tag = serializers.SerializerMethodField()
```

Comment:
I'm curious why this is a `SerializerMethodField`. I think it's possible to write it as `tag = CharField(source='make_tag', read_only=True)`

While that may not be obvious or intuitive to write, I would argue it's clearer to read. In general, the `SerializerMethodField` is useful when the serializer itself needs to do some complex computation, like you've done with get_user_fields. But this is just a straight call to the model object, which a declarative field can do.